src/config/index.ts

```diff
@@ -99,7 +99,9 @@ export const getNetworks = (): NetworkInfo[] => {
   }))
 }
 
-export const getClientGatewayUrl = (): string => getConfig().clientGatewayUrl
+export const getRootClientGatewayUrl = (): string => getConfig().clientGatewayUrl
+
+export const getClientGatewayUrl = (): string => `${getRootClientGatewayUrl()}/chains/${getNetworkId()}`
```

Comment:
We should eventually move this logic to the SDK and pass only the chainId and env as params. But for now it's a good solution to avoid many changes.

Reply:
I couldn't avoid following this suggestion now :smile: 

https://github.com/gnosis/safe-react/pull/2616/commits/fc37d0c6b9c950c565049b3db6adea078b8ee534